### DISCUSSION TITLE
Add MongoDB Support

### DISCUSSION THREAD

**Discussion by Jaydar** [tokio-rs/toasty#39] [Polls]
What we need more is that MongoDB, DynamoDB and Cassandra have fewer users compared to the former.

**Comment by carllerche** [comment]
MongoDB support should be relatively easy to add. DynamoDB/Cassandra were mostly picked as the first targets because of the limited feature set (if it works w/ those, it should work w/ most). There are a few significant driver related re-factors in progress, but after that toasty should be in a good place to add more DB support.
